system_baseline/app.py

```diff
@@ -37,16 +37,25 @@ def create_connexion_app():
     flask_app.url_map.strict_slashes = True  # it needs to be set before add_api call
 
     connexion_app.add_api(
-        "api.spec.yaml", arguments=openapi_args, strict_validation=True, validate_responses=True
+        "api.spec.yaml",
+        arguments=openapi_args,
+        strict_validation=True,
+        validate_responses=True,
+        resolver_error=501,
```

Comment:
I thought that we'd stick with the 405, that is handled by the framework. No need to override this.